### DISCUSSION TITLE
abou default resolvers

### DISCUSSION THREAD

**Discussion by 666asd** [projectdiscovery/nuclei#5296] [Q&A]
### Please describe your feature request:
Can you add a command line parameter that can be resolved directly using the system configured DNS server without using the default DNS server. 

When the default DNS cannot be accessed in the internal network, DNS resolution will be slow.

**Comment by GeorginaReeder** [comment]
Thanks for your suggestion @666asd , we'll take a look into this!

**Comment by ehsandeep** [comment]
@666asd see the below option

```
nuclei -h sr

Usage:
  nuclei [flags]

Flags:
   -sr, -system-resolvers  use system DNS resolving as error fallback
```

**Reply by 666asd** [reply]
Thanks, I tried that option.This option does not seem to have any effect. Whether this option is used or not, the default DNS is invoked for resolution, and then the system DNS is used. I want to use the system dns for resolution without using the default dns built in nuclei.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
